docs/guides/auth.md

```diff
@@ -326,3 +343,31 @@ $opts = [
 ];
 $client = new helloworld\GreeterClient('greeter.googleapis.com', $opts);
 ```
+
+###Authenticating with Google (Python)
+
+####Base case - No encryption/authentication
+```python
+channel = implementations.insecure_channel('localhost', 50051)
+stub = helloworld_pb2.beta_create_Greeter_stub(channel)
+...
+```
+
+###Authenticate using OAuth2 token (legacy approach)
```

Comment:
I think this did fall through the cracks. For some reason I thought it was done, but in retrospect I remember reviewing the Go, Ruby and PHP implementations but not this one. We did have a tracking bug https://github.com/google/oauth2client/issues/252, but it did not get assigned or implemented.